src/index.js

```diff
@@ -223,6 +223,10 @@ const getStatus = id => {
   return status || unknownStatus
 }
 
+// Not all the fields being searched on should be visible but need
+// to be on the DOM in order for listjs to pick them up for search
+const hiddenSearchFields = ['address', 'accepting', 'notAccepting', 'notes']
```

Comment:
```suggestion
const hiddenSearchFields = ['address', 'accepting', 'notAccepting', 'notes', 'seekingVolunteers']
```
Need to include "seeking volunteers" in our search fields because users expect to find detailed information about types of volunteer work here. This field has information like 'bilingual' or 'sign up here'